fluent-package/yum/install-test.sh

```diff
@@ -83,8 +83,8 @@ for conf_path in /etc/td-agent/td-agent.conf /etc/fluent/fluentd.conf; do
     fi
 done
 
-(! getent passwd fluentd >/dev/null)
-(! getent group fluentd >/dev/null)
+getent passwd fluentd >/dev/null
+getent group fluentd >/dev/null
```

Comment:
This fails for the `v5` and `lts` tests until the v5.0.2 is released.
How about this?

```suggestion
case $1 in
  local)
    getent passwd fluentd >/dev/null
    getent group fluentd >/dev/null
    ;;
  *)
    # TODO: Remove this branch after the following fix is applied to the latest release.
    # https://github.com/fluent/fluent-package-builder/pull/598
    (! getent passwd fluentd >/dev/null)
    (! getent group fluentd >/dev/null)
    ;;
esac
```

Reply:
Wait, I'll revert it.

Reply:
yum/install-test.sh always tests locally build package.
Thus, it doesn't matter that is released or not.

Reply:
I'm sorry.
I was suggesting the wrong file.